app/views/people/new.html.erb

```diff
@@ -4,13 +4,12 @@
 
 <%= form_for @person do |f| %>
     <%= f.label :name, "What's your name?" %>
-
     <%= f.text_field :name, placeholder: "Pietje Precies" %><br />
       <p></p>
-    <%= f.label :length, "What is your height" %>
+    <%= f.label :length, "What is your height?" %>
```

Comment:
`length` is often used by Ruby and Ruby on Rails classes. Array for instance uses `length` to get the _length_ of the Array. http://apidock.com/ruby/Array/length

To avoid confusion later it might be wise to change the name of this field to something else.